docs/madr/decisions/030-extended-global-insights.md

```diff
@@ -51,8 +51,6 @@ should look like this:
 
 ```json
 {
-   "resources": ...,
-   // <== this stays the same
```

Comment:
Could we add simple sentence that it will be removed and why we decided to do it?